api/open-positions.ts

```diff
@@ -0,0 +1,47 @@
+import Airtable from "airtable";
+import { NowRequest, NowResponse } from "@now/node";
+
+interface JobRecord {
+  "Stav": "Aktivní" | "Obsazeno";
+  "Úkol/Role": string;
+  "Odkaz na celou příležitost": string | null;
+  "Detaily": string | null;
+  "Časová náročnost": string | null;
+  "Urgentní poptávka": boolean;
+}
+
+interface Position {
+  title: string;
+  isOpen: boolean;
+  isUrgent: boolean;
+  detailUrl: string | null;
+  description: string | null;
+  timeRequirements: string | null;
+}
+
+export default async (req: NowRequest, res: NowResponse) => {
+  try {
+    const apiKey = process.env.AIRTABLE_API_KEY as string;
+    const table = new Airtable({ apiKey }).base("apppZX1QC3fl1RTBM")(
```

Comment:
Má tu být base? Není to taky secret?

Reply:
Nedokážu posoudit ani opravit (do secrets nemám přístup).

Reply:
Dobrý postřeh, díky. Defenzivnější by bylo to schovat, ale už jsme to stejně leaknuli i v jiných skriptech a přegenerovat to nejde, takže asi můžem nechat.